### PR TITLE
Maya/161386 create usher button

### PR DESCRIPTION
**Pull request by mmmmmmaya**
### Summary
- **What:**
  - Update text in tooltip
  - Add a menu to the `Create tree` button, instead of opening Nextstrain modal directly
  - Move icon files to the icon folder ???
  - Add a new feature flag for usher work
  - Create a [ff hidden] menu option for usher trees
  - There were some uncommitted package-lock changes
- **Ticket:** [[sc-161386]](https://app.shortcut.com/genepi/story/161386)
- **Env:** https://ushermenu-frontend.dev.genepi.czi.technology

### Demos
Screen Shot 2021-09-30 at 1 32 10 PM
Screen Shot 2021-09-30 at 1 37 29 PM
Screen Shot 2021-09-30 at 1 37 16 PM

### Notes
The design shows a particular alignment for the menu. Unfortunately SDS' `Menu` didn't previously allow for custom menu alignment. I have added that functionality [here](https://github.com/chanzuckerberg/sci-components/pull/67), and will bump the `czifui` package version in package.json whenever there is a new SDS release.

### Checklist
- [x] I merged latest `trunk`
- [x] I manually verified the change
- [x] I added labels to my PR
- [ ] I tested in multiple browsers
- [ ] I added relevant unit tests
- [ ] I have notified others of changes they need to make locally (migrations, jobs, package updates, etc)

### CODE REVIEW

**Review by vincent-czi — APPROVED** [review]
Looks good! Nice work

**Comment by vincent-czi** [comment]
My only note is that you might want to get Product/Design approval on the feature flag aspect since it's still a change to how the app behaves even when usher flag is false. Instead of clicking it and going directly into the modal on click, it will bring up a 1-item menu as an intermediate step. I think it's fine, but I'd verify that with them that it's okay for the next couple weeks to look like that. Since we now have feature flags, I assume we'll actually have this code running on Prod pretty soon, so approval seems necessary to me.

**Comment by mmmmmmaya** [comment]
Will merge pending approval https://czi-sci.slack.com/archives/C01HYJYA50W/p1633039550209500

**Pull request merged**
